cmake/config-unix/pyconfig.h.in

```diff
@@ -1160,7 +1160,13 @@
 #cmakedefine _GNU_SOURCE 1
 
 /* This must be defined on some systems to enable large file support. */
-#cmakedefine _LARGEFILE_SOURCE @_LARGEFILE_SOURCE@
+#cmakedefine _PY_LARGEFILE_SOURCE @_LARGEFILE_SOURCE@
+#if _LARGEFILE_SOURCE != _PY_LARGEFILE_SOURCE
+# error "_LARGEFILE_SOURCE is inconsistently defined"
```

Comment:
is it necessary to make this an error? the check should never be anything other than `ifdef`/`ifndef`.